task/build-paketo-builder-oci-ta/0.1/build-paketo-builder-oci-ta.yaml

```diff
@@ -232,12 +228,36 @@ spec:
         )
 
         if [ "${HERMETIC}" == "true" ]; then
-          for image in $BASE_IMAGES; do
+          UNSHARE_ARGS+=("--net")
+
+          for image in ${BASE_IMAGES}; do
             echo "Pull the base image: $image using unshare"
             unshare -Ufp --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- buildah pull $image
           done
+
+          for image in ${BUILDPACKS_IMAGES}; do
+            echo "Pull the buildpacks image: $image using unshare"
+            unshare -Ufp --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- buildah pull $image
+          done
+
+          echo "Fetch lifecycle and patch the ${PACK_BUILDER_FILE} file ..."
+          LIFECYCLE_VERSION=$(tomljson ${PACK_BUILDER_FILE} | jq -r '.lifecycle.version')
+          unshare -Ufp --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- \
+            curl -skL https://github.com/buildpacks/lifecycle/releases/download/v${LIFECYCLE_VERSION}/lifecycle-v${LIFECYCLE_VERSION}+linux.x86-64.tgz --output source/lifecycle-v${LIFECYCLE_VERSION}+linux.x86-64.tgz
+
+          echo "Patch the builder toml file to use the local PATH of the lifecycle tar.gz file"
+          tomljson ${PACK_BUILDER_FILE} > source/builder.json
+          jq 'del(.lifecycle.version) | .lifecycle.uri = "lifecycle-v'"${LIFECYCLE_VERSION}"'+linux.x86-64.tgz"' source/builder.json > source/new-builder.json
+
+          PACK_BUILDER_FILE="source/new-builder.toml"
+          pip install tomli_w
```

Comment:
This should be installed in paketo-container and not here during runtime

Reply:
Fixed. See: 0c1e1ae